qcodes/instrument_drivers/Keysight/N52xx.py

```diff
@@ -344,6 +371,15 @@ def __init__(self,
                            get_parser=int,
                            set_cmd="CALC:PAR:MNUM {}",
                            vals=Numbers(min_value=1, max_value=24))
+
+        self.add_parameter(
+            "trigger_source",
+            get_cmd="TRIG:SOUR?",
+            set_cmd="TRIG:SOUR {}",
+            vals=Enum("EXT", "IMM", "INT", "MAN"),
+            set_parser=lambda value: "IMM" if value is "INT" else value
```

Comment:
I was thinking about that, but val mapping will not work in this case, as the mapping is not one to one. The point of this was that selecting "INT", which stands for "internal" should be an alias for "IMM" (immediate). I find the former a much more logical name, although that might be subjective. 

The point is, both "IMM" and "INT" mean the same thing and since with val mapping you use a dictionary and dictionaries have unique keys, this is impossible to accomplish through that mechanism.